Caltech-IPAC/firefly
# Firefly-386: add orbital path support to drawing

Firefly-386: add orbital path support to drawing

  - also implements Firefly-486: file name now in tabs for upload
  - clean up of PlotHipsTask.js
  - fixes firefly-485
  - added a test to demonstrate orbital path

_Jira:_
https://jira.ipac.caltech.edu/browse/FIREFLY-386
https://jira.ipac.caltech.edu/browse/FIREFLY-486



#### To test:
- https://irsawebdev9.ipac.caltech.edu/firefly-386-oribital-path/firefly/
- look at the orbital path test: https://irsawebdev9.ipac.caltech.edu/firefly-386-oribital-path/firefly/test/tests-main.html

_or_
- using firefly, upload the file attach in the ticket

_note_: if the table is sorted or filtered, the user can no longer click on a point. Since the orbital path no longer matches the table.


#### To test Firefly-486:

- upload a catalog table using the catalog upload, the table name should be in the tab.
